src/postgres/db.js

```diff
@@ -0,0 +1,68 @@
+const { Pool } = require("pg");
+require("dotenv").config({
+  override: true,
+  path: "././development.env",
+});
+
+const pool = new Pool({
+  user: process.env.USER,
+  host: process.env.HOST,
+  database: process.env.DATABASE,
+  password: process.env.PASSWORD,
+  port: process.env.PORT,
+});
+
+console.log(
+  process.env.USER,
+  process.env.HOST,
+  process.env.DATABASE,
+  process.env.PASSWORD,
+  process.env.PORT
+);
+
+
+const urlPool = new Pool({
+  user: process.env.USER,
+  host: process.env.HOST,
+  database: process.env.DATABASE,
+  password: process.env.PASSWORD,
+  port: process.env.PORT,
+});
+
+const { Client } = require("pg");
+
+//IMPORTANT: tried to connect to dedicated url_shortener db via .env file - keeps connecting to default postgres db
+
+const client = new Client({
+  user: process.env.USER,
+  host: process.env.HOST,
+  database: "url_shortener",
+  password: process.env.PASSWORD,
+  port: process.env.PORT,
+});
+
+client.connect();
+client.query(
+  `
+
+  CREATE TABLE IF NOT EXISTS public.url_shortener
+  (
+      id integer NOT NULL DEFAULT nextval('url_shortener_id_seq'::regclass),
+      longurl character varying(255) COLLATE pg_catalog."default",
+      shorturl character varying(255) COLLATE pg_catalog."default",
+      issueddate timestamp with time zone,
+      CONSTRAINT url_shortener_pkey PRIMARY KEY (id)
+  )
+
+  TABLESPACE pg_default;
+
+  ALTER TABLE IF EXISTS public.url_shortener
+      OWNER to postgres;
```

Comment:
These are multiple queries into 1 query functional call. It might be not accepted. 

Also, the alter owner and tablespace queries are not suited for a migration script. They need to be configured beforehand with the setup of the database. That's why you need to make sure to have a dedicated dev user per db (ex. url-shortener-db-user) for the db, that has enough rights and permissions to create, alter, write, and read to the tables of the specific db. 

import { Client } from 'pg'
const client = new Client({..})
 
await client.connect()
 
const createResult = await client.query('CREATE TABLE')
console.log(result)
 
await client.end()

Reply:
Do not use 'postgres' user, except if you have your database running in a docker container.